Makefile

```diff
@@ -110,10 +118,15 @@ cephcsi: check-env
 	if [ ! -d ./vendor ]; then (go mod tidy && go mod vendor); fi
 	GOOS=linux go build -mod vendor -a -ldflags '$(LDFLAGS)' -o _output/cephcsi ./cmd/
 
+e2e.test: check-env
```

Comment:
this can be renamed to compile-e2e?

Reply:
it can, but `e2e.test` is the standard name of the executable that `go test` generates. I do not think it improves clarity by adding an indirection

Reply:
it will generate `pkg.test` where pkg is e2e here sn't it? compile-e2e is kind of more meaning full that e2e tests are compiled into a binary

Reply:
You'll get a `pkg.test` you run `go test pkg`, or `go test ./internal` will create a `internal.test` executable. The name of the executable depends on the go-package that you want to test.